### PR TITLE
New fern branch

### PR DESCRIPTION
**Pull request by dlayton66**
## User Story(s) affected
- Create new fern

## Describe Changes
- Finishes create fern

## Requested Reviewer(s)
Anthony O.
Sam

### CODE REVIEW

File: spec/vcr/FernService/_create_fern/creates_a_fern.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+http_interactions:
+- request:
+    method: post
+    uri: http://localhost:5000/api/v1/users/110920554030325122207/ferns
+    body:
+      encoding: UTF-8
+      string: name=The+Big+Pepperoni&preferred_contact_method=text&shelf=Family
+    headers:
+      Fern-Key:
+      - FErnCultForLyfe
```
Comment (JustJakeSeymour) [inline spec/vcr/FernService/_create_fern/creates_a_fern.yml]: do we need to be keeping this hidden? AKA should the vcr folder not be here?

Comment (ajongaro) [reply spec/vcr/FernService/_create_fern/creates_a_fern.yml]: Yes, this Fern-Key should not be on GH imo.

Comment (dlayton66) [reply spec/vcr/FernService/_create_fern/creates_a_fern.yml]: Should be hidden with the latest push now :)

**Review by sambcox — APPROVED** [review]
All looks good!

**Pull request merged**